src/sentry/api/endpoints/release_thresholds/utils.py

```diff
@@ -60,3 +62,61 @@ def get_errors_counts_timeseries_by_project_and_release(
     )["data"]
 
     return data
+
+
+from django.utils.datastructures import MultiValueDict
+from rest_framework.request import Request
+
+from sentry import release_health
+from sentry.api.utils import handle_query_errors
+from sentry.models.organization import Organization
+from sentry.snuba.sessions_v2 import QueryDefinition
+
+# In minutes
+TWO_WEEKS = 20160
+ONE_WEEK = 10080
+TWENTY_FOUR_HOURS = 1440
+
+
+def fetch_sessions_data(
+    request: Request,
+    organization: Organization | RpcOrganization,
+    params: FilterParams,
+    end: datetime,
+    start: datetime,
+):
+    """
+    This implementation was derived from organization_sessions GET endpoint
+    NOTE: Params are derived from the request query and pulls the relevant project/environment objects
+    TODO: make fetch generic for other session types
+    TODO: capture potential issues with `interval` returning too many results
+    """
+    with handle_query_errors():
+        request_get: dict[str, Any] = request.GET
+        query_params: MultiValueDict[str, Any] = MultiValueDict(request_get)
+        query_params.setlist("groupBy", ["project", "release", "session.status", "environment"])
+        query_params.setlist("field", ["sum(session)"])  # alternatively count_unique(user)
```

Comment:
should you pass `CRASH_SESSIONS_DISPLAY` to determine if it's `sum(session)` or `count_unique(user)`?